website/redirects.js

```diff
@@ -149,7 +149,7 @@ const Redirects=[
   },
   {
     "from": "/docs/latest/development/extensions-core/namespaced-lookup.html",
-    "to": "/docs/latest/development/extensions-core/lookups-cached-global"
```

Comment:
we'd want this to be an additional entry for the `from` wouldn't we? Right now this redirect goes from really old page to the current page. Or it sould replace the old `from` entry